src/components/mainLayout/signUpForm/signUpForm.tsx

```diff
@@ -34,6 +33,9 @@ const SignUpForm = () => {
 	});
 
 	const router = useRouter();
+	const errorStatusMin = 500;
+	const errorStatusMax = 600;
```

Comment:
can you move it to exported enum in src/helpers/ApiResponseCode.ts and use here?